### ISSUE TITLE
On-the-fly brute-force selector guesser on failed user search, stream results to frontend

### ISSUE THREAD

**Issue by foolo** [zkemail/archive.prove.email#90]
New issue from this comment https://github.com/zkemail/archive.prove.email/issues/70#issuecomment-2057278860

When a user searches for a domain which we don't have in the database, show a message such as _"No selector found for example.com. Trying common selectors..."_ and then, if a result was found, update the list on the webpage.

Steps needed:
- Do parallel brute-force check of the list of about ~2000 frequently used selectors. Needs to be routed via website server.
- Need another, dedicated server (or 3rd party service) to perform it in parallel.
- Implement a mechanism to propagate the result back to frontend.

Cost:  > 2 weeks of work. 

Cost/benefit:
My personal take is that this is an expensive feature, that has both an upfront cost and a long-term maintenance cost. I agree that IF a selector is actually found, then it would look really cool! But I think that in 99 % of the cases it will render 0 results, and possibly annoy the user more than it helps, because the user will wait on the progress just to see "nothing found".

**Comment by Divide-By-0** [comment]
Here's one cool code snippet where someone analyzed common patterns in DLIM selectors and gave Perl to possibly autocomplete them, even weird date ones: https://github.com/ryancdotorg/dkimscan/blob/master/dkimscan.pl
